### PR TITLE
delete paddle api

### PR DESCRIPTION
**Pull request by JepsonWong**
delete paddle api.

* paddle.nn
  * paddle.nn.functional
  * paddle.nn.layer
* paddle.optimizer
* paddle.meric
* paddle.framework
* paddle.io
* paddle.imperative

### CODE REVIEW

**Review by zhwesky2010 — DISMISSED** [review]
remove paddle.nn.API in release/1.8, delete unittest

**Review by zhwesky2010 — DISMISSED** [review]

**Review by zhwesky2010 — DISMISSED** [review]
move unittest

File: python/paddle/fluid/tests/unittests/test_activation_op.py [hunk 1]
```diff
@@ -737,6 +735,9 @@ def setUp(self):
         self.outputs = {'Out': out}
 
     def test_check_grad(self):
+        import paddle
```
Comment (phlrain) [inline python/paddle/fluid/tests/unittests/test_activation_op.py]: 这个是不是跟21行的重复

Comment (JepsonWong) [reply python/paddle/fluid/tests/unittests/test_activation_op.py]: done

**Review by zhwesky2010 — APPROVED** [review]

**Review by raindrops2sea — APPROVED** [review]
LGTM

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by swtkiwi — APPROVED** [review]
LGTM

**Pull request merged**